ground/gcs/src/plugins/boards_brainfpv/brainre1.cpp

```diff
@@ -200,6 +200,72 @@ Core::IBoardType::InputType BrainRE1::getInputType()
         break;
     }
 
+    switch (settings.SerialPort) {
+    case HwBrainRE1::SERIALPORT_SBUS:
+        return INPUT_TYPE_SBUS;
+    case HwBrainRE1::SERIALPORT_SBUSNONINVERTED:
+        return INPUT_TYPE_SBUSNONINVERTED;
+    case HwBrainRE1::SERIALPORT_DSM:
+        return INPUT_TYPE_DSM;
+    case HwBrainRE1::SERIALPORT_HOTTSUMD:
+        return INPUT_TYPE_HOTTSUMD;
+    case HwBrainRE1::SERIALPORT_HOTTSUMH:
+        return INPUT_TYPE_HOTTSUMH;
+    case HwBrainRE1::SERIALPORT_IBUS:
+        return INPUT_TYPE_IBUS;
+    case HwBrainRE1::SERIALPORT_SRXL:
+        return INPUT_TYPE_SRXL;
+    case HwBrainRE1::SERIALPORT_TBSCROSSFIRE:
+        return INPUT_TYPE_TBSCROSSFIRE;
+    default:
+        break;
+    }
+
+    switch (settings.MultiPortSerial) {
+    case HwBrainRE1::MULTIPORTSERIAL_SBUS:
+        return INPUT_TYPE_SBUS;
+    case HwBrainRE1::MULTIPORTSERIAL_SBUSNONINVERTED:
+        return INPUT_TYPE_SBUSNONINVERTED;
+    case HwBrainRE1::MULTIPORTSERIAL_DSM:
+        return INPUT_TYPE_DSM;
+    case HwBrainRE1::MULTIPORTSERIAL_HOTTSUMD:
+        return INPUT_TYPE_HOTTSUMD;
+    case HwBrainRE1::MULTIPORTSERIAL_HOTTSUMH:
+        return INPUT_TYPE_HOTTSUMH;
+    case HwBrainRE1::MULTIPORTSERIAL_IBUS:
+        return INPUT_TYPE_IBUS;
+    case HwBrainRE1::MULTIPORTSERIAL_SRXL:
+        return INPUT_TYPE_SRXL;
+    case HwBrainRE1::MULTIPORTSERIAL_TBSCROSSFIRE:
+        return INPUT_TYPE_TBSCROSSFIRE;
+    default:
+        break;
+    }
+    
+    if (settings.MultiPortMode == HwBrainRE1::MULTIPORTMODE_DUALSERIAL4PWM)
```

Comment:
Not sure if this check is needed?
Where can I see what's the default value of settings.MultiPortSerial2, if the MultiPortMode is set to standard?